### ISSUE TITLE
renewal broken on centos 6.7

### ISSUE THREAD

**Issue by SwartzCr** [letsencrypt/letsencrypt#2430]
Processing /etc/letsencrypt/renewal/noah2.isnot.org.conf
2016-02-10 19:46:57,017:WARNING:letsencrypt.cli:Attempting to renew cert from /etc/letsencrypt/renewal/noah2.isnot.org.conf produced an unexpected error: Directory '/etc/httpd/sites-enabled' does not exist. Please ensure that the values for --apache-handle-sites and --apache-server-root are correct for your environment.. Skipping.

All renewal attempts failed. The following certs could not be renewed:
  /etc/letsencrypt/live/noah2.isnot.org/fullchain.pem (failure)

It seems that we're not taking system specific paths into account well enough, leading us to look for sites-enabled on centos which doesn't typically have that

**Comment by SwartzCr** [comment]
[[webroot_map]]
[root@noah2 letsencrypt]# cat /etc/letsencrypt/renewal/noah2.isnot.org.conf | grep apache_
apache_enmod = None
apache_dismod = None
apache_handle_modules = False
apache_handle_sites = False
apache_challenge_location = /etc/httpd/conf.d
apache_server_root = /etc/httpd
apache_init_script = None
apache_ctl = None
apache_le_vhost_ext = -le-ssl.conf
apache_vhost_root = /etc/httpd/conf.d

```
1296     def is_site_enabled(self, avail_fp):                                         
1297         """Checks to see if the given site is enabled.                           
1298                                                                                  
1299         .. todo:: fix hardcoded sites-enabled, check os.path.samefile            
1300                                                                                  
1301         :param str avail_fp: Complete file path of available site                
1302                                                                                  
1303         :returns: Success                                                        
1304         :rtype: bool                                                             
1305                                                                                  
1306         """                                                                      
1307                                                                                  
1308         enabled_dir = os.path.join(self.parser.root, "sites-enabled")            
1309         if not os.path.isdir(enabled_dir):                                       
1310             error_msg = ("Directory '{0}' does not exist. Please ensure "        
1311                          "that the values for --apache-handle-sites and "        
1312                          "--apache-server-root are correct for your "            
1313                          "environment.".format(enabled_dir))  
```

**Comment by SwartzCr** [comment]
guess we should address this todo in configurator.py

**Comment by SwartzCr** [comment]
should be fixed in #2431
